File: Algo/hr_implement/3/findDigits_divisior.js
function findDigits(n) {
    let count = 0;
    n.toString().split('').forEach(num => {
        if (num != '0' && n % parseInt(num) == 0) {
            count++;
        }
    });
    return count;
}

console.log(findDigits(1012));

File: Algo/hr_implement/3/findDigits_divisior.cs
using System;
using System.Linq;
using System.Collections;

public class Test
{
    public void TestFun()
    {
        int n = 1012, count = 0;
        foreach (var x in Convert.ToString(n))
        {
            if (x != '0' && n % Convert.ToInt16(Convert.ToString(x)) == 0)
            {
                count++;
            }
        }
        Console.WriteLine(count);
    }
}